src/Directory.build.props

```diff
@@ -1,26 +1,56 @@
 <Project>
-  <PropertyGroup>
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+  <!-- Only the first Directory.Build.props is automatically loaded. The rest need to be loaded manually -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)..\'))" />
 
-    <LangVersion>7.2</LangVersion>
-    <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)/../build/MSAL.snk</AssemblyOriginatorKeyFile>
-    <CodeAnalysisRuleSet>$(MSBuildThisFileDirectory)/../build/SolutionWideAnalyzerConfig.ruleset</CodeAnalysisRuleSet>
+  <PropertyGroup Label="Versioning">
+    <!--This should be passed from the VSTS build-->
+    <MsalClientSemVer Condition="'$(MsalClientSemVer)' == ''">4.4.0-localbuild</MsalClientSemVer>
+    <!--This will generate AssemblyVersion, AssemblyFileVersion and AssemblyInformationVersion-->
+    <Version>$(MsalClientSemVer)</Version>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Packaging">
```

Comment:
@jmprieur - you might want to review the package metadata (this is for all the nuget packages)